gke-deploy/services/exec.go

```diff
@@ -12,14 +12,14 @@ import (
 	"github.com/pkg/errors"
 )
 
-func runCommandWithStdinRedirection(printCommand bool, name, input string, args ...string) (string, error) {
+func runCommandWithStdinRedirection(ctx context.Context, printCommand bool, name, input string, args ...string) (string, error) {
 	if printCommand {
 		fmt.Printf("\n--------------------------------------------------------------------------------\n")
 		fmt.Printf("> Running command\n\n")
 		fmt.Printf("   %s %s < %s\n", name, strings.Join(args, " "), input)
 		fmt.Printf("\n--------------------------------------------------------------------------------\n\n")
 	}
-	cmd := exec.Command(name, args...)
+	cmd := exec.CommandContext(ctx, name, args...)
 	cmd.Stderr = os.Stderr
 	// Example taken from https://golang.org/src/os/exec/example_test.go
 	stdin, err := cmd.StdinPipe()
```

Comment:
Awesome. I think runCommandWithContext can be renamed to runCommand, now that they all require context

Reply:
Done.